prime-router/metadata/schemas/covid-19.schema

```diff
@@ -405,35 +405,38 @@ elements:
 
   - name: patient_age
     type: NUMBER
+    cardinality: ZERO_OR_ONE
     natFlatFileField: Patient_age
     hhsGuidanceField:
     hl7Field: AOE
     hl7AOEQuestion: 30525-0
 
   - name: patient_age_units
     type: CODE
+    cardinality: ZERO_OR_ONE
     valueSet: patient_age_units
     natFlatFileField: Patient_age_units
     hhsGuidanceField:
     documentation: Always filled when `patient_age` is filled
 
   - name: patient_city
     type: CITY
-    cardinality: ONE
+    cardinality: ZERO_OR_ONE
     natFlatFileField: Patient_city
     hhsGuidanceField:
     hl7Field: PID-11-3
     documentation: The patient's city
 
   - name: patient_county
-    type: TABLE
+    type: TABLE_OR_BLANK
     table: fips-county
```

Comment:
Cardinality of ZERO_OR_ONE didn't work for patient_county, I believe because the mapping ignores this.

So, I hackishly added an OR_BLANK type, which has almost the same semantics.  Per other discussions, I think this is ripe for simplification.